source/includes/v1/query_language_and_geo_filtering.md

```diff
@@ -1,11 +1,11 @@
 # Query Language and Geo Filtering
 
-Filtering features are built in the core of Opendatasoft API engine. Many of the previously listed APIs can take as a
-parameter filters for constraining the list of returned datasets or records.
+Filtering features are built in the core of the Opendatasoft API engine. Many of the previously listed APIs can take as
+parameters filters for constraining the list of returned datasets or records.
```

Comment:
Maybe worth it to re-organize the sentence or add commas as I had to read it 3 times to get the sence xD

Reply:
It's true that this sentence is not crystal clear. I added a better [suggestion]( https://github.com/opendatasoft/ods-documentation-explore-api/pull/78/files/f400478e6c7091431b86c26f9b67fc9c66b6edc6#r576330398).